### DISCUSSION TITLE
How to apply a pressure load on the surface of a shell element ？

### DISCUSSION THREAD

**Discussion by tttsss-01** [idaholab/moose#23711] [Q&A Modules: Solid mechanics] [answered]
I want to use shell elements to simulate the pressure load inside a cylinder, how to do it? 
Pressure in BCs doesn't seem to be able to handle this.

**Comment by GiudGiud** [comment]
@neuphris @recuero on shell elements

**Comment by hugary1995** [comment, marked as answer]
You need to implement a custom body force kernel for this. PR welcome.
